.gitignore

```diff
@@ -2,6 +2,7 @@
 /local_staging
 /data
 /dask-worker-space
+/~
```

Comment:
This was caused by a bug and I _think_ I fixed it in one of my PRs to this branch. Can remove.